<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeatherWindow</class>
 <widget class="QDialog" name="WeatherWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="resultLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>320</y>
     <width>341</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="getWeatherButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Узанть погоду</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cityEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>341</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Введите название города</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
